backend/src/main/java/app/domain/game/GameService.java

```diff
@@ -46,51 +44,44 @@ public UUID joinToGame(String playerName) {
     }
 
     public void startGame() {
-        timer.start();
-        game.activatePlayers();
         startRound();
-        publisher.publishEvent(new GamePlayersChanged(this, getPlayers()));
+        game.setGameTimeStamp(Instant.now().toEpochMilli());
```

Comment:
toEpochMilli - "the number of milliseconds since the epoch of 1970-01-01T00:00:00Z.". to już lepiej zrób LocalDateTime.now()